fhir/doc/tutorial.md

```diff
@@ -0,0 +1,147 @@
+# Leading Edge Acceleration Projects (LEAP)
+## _Guide To Using [Layered Schemas Architecture (LSA)](https://github.com/cloudprivacylabs.com/lsa)_
+https://github.com/cloudprivacylabs.com/lsa
+#### From the [Layered Schemas Architecture](https://layeredschemas.org/) Site:
+#
+> Layered Schema Architecture (LSA) enables semantic interoperability between heterogeneous systems. LSA uses a schema base (such as FHIR schemas for health data) to define data structures, and overlays to add semantics, contextual metadata, and processing directives. A schema variant is the composition of the schema base and use-case specific overlays. Different schema variants can be used to ingest data from or export data to disparate systems. Each variant encodes source specific metadata and rules to ingest data into a knowledge graph, or target specific metadata and rules to translate the knowledge graph into another format.
+
+## Goal:
+By the end of this tutorial, you will have an understanding of what Layered Schemas Architecture (LSA) is and how to use its technology given an _OMOP_ concept. For our example, we will be using the [__OMOP__](https://www.ohdsi.org/data-standardization/) concepts dataset - you can find all Omop concepts here: https://athena.ohdsi.org/search-terms/start
+
+## Prerequisites:
+### If you do not have the following databases installed below, follow the direct page's instructions to set them up.
+- [`PostgreSQL`](https://www.postgresql.org/)
+-- can be replaced by your favorite table based relational database - MySQL, MariaDB, Microsoft SQL Server, Oracle Database, etc.
+- [`Neo4j`](https://neo4j.com/)
+- [`Layered Schema Architecture`](https://github.com/cloudprivacylabs.com/lsa)
+    - install or build binary
+- [`LSA Neo4j`](https://github.com/cloudprivacylabs.com/lsa-neo4j)
+    - install or build binary
+
+## Setting Up:
+Now that you have both databases installed on your machine, let's first download the dataset you wish to use from https://athena.ohdsi.org/vocabulary/list. Once that is downloaded, the next step is to import that valueset into PostgreSQL (or your RDBMS of choice). 
+
+## Importing Concepts to PostgreSQL
+The first step to importing data to our database is creating the matching columns of our downloaded valueset.
+Go to https://athena.ohdsi.org/search-terms/start site and click on the domain dataset of your choice. After clicking on the domain, click on any row - this should bring you to a page that looks the following:
+
+![athena](tutorial_athena.png)
+
+In this picture, the details list for the drug Promedrol are composed of the rows:
+- Domain ID
+- Concept Class ID
+- Vocabulary ID
+- Concept ID
+- Concept code
+- Validity
+- Concept
+- Valid start
+- Valid end
+
+In PostgreSQL, we must create a table with columns that match the rows of our concepts.
+```
+CREATE TABLE concepts (
```

Comment:
Is there a recommendation for this? All cols 255 is too much

Also, add that it should be indexed:

concept_id unique index
Non-unique indexes: 
concept_code
concept_name